tests/k6/src/test-delete-file.ts

```diff
@@ -0,0 +1,49 @@
+import {sleep, check} from 'k6';
+import {Options} from "k6/options";
+import {defaults, api} from "./lib";
+
+const files = {
+    'kb_50.jpg': open('./_files/kb_50.jpg', 'b'),
+}
+
+export let options: Options = {
+    insecureSkipTLSVerify: true,
+    iterations: 100,
+    vus: 100,
+};
+
+export const setup = (): void => {
+    console.log("setup for delete")
+    for (let vu=1; vu <= options.vus; vu++) {
+      for (let iter=0; iter < options.iterations/options.vus; iter++) {
+        const res = api.uploadFile(defaults.accounts.einstein, files['kb_50.jpg'], `deletefile_${vu}_${iter}.jpg`)
+        check(res, {
+          'status is 201': () => res.status === 201,
+        })
+      }
+    }
+    console.log("uploaded test files")
+    sleep(1)
+}
+
+export default () => {
+    const res = api.deleteFile(defaults.accounts.einstein, `deletefile_${__VU}_${__ITER}.jpg`)
+    check(res, {
+        'status is 204': () => res.status === 204,
+    });
+    sleep(1);
```

Comment:
Someone educate me. Why do we `sleep(1)` here after doing the API action and checking the status?

Reply:
This is the main test function which is executed for each virtual user, since in this test we are running 100 iteration with 100 vus, each virtual users will only run this function once, but if we were to run with less number of virtual users this will wait between requests for each virtual users for one second.

Reply:
OK, so this controls the "work rate" of a virtual user. (rather than it being an externally-controllable thing)